core/embed/extmod/trezorobj.h

```diff
@@ -27,6 +27,12 @@
 #error Use MPZ for MicroPython long int implementation.
 #endif
 
+#ifdef TREZOR_EMULATOR
+#define STATICTYPE
```

Comment:
Is there a reason why not to just override STATIC completely for emulator?

Reply:
i don't understand reasons for `STATIC` in the first place so I made the change as non-invasive as possible.

of course if you think we can switch off `STATIC` that works too

Reply:
for that matter, it would also remove the need for the micropython hack mentioned above, which would be good.

question is, won't removing STATIC get the emulator farther from the real thing in a noticeable way?